change-log.md

```diff
@@ -553,6 +553,8 @@
 
 ### Fixed:
 
+-   Fixed: [178d520](https://github.com/veeupup/openraft/commit/178d520e3db47b361582fe4aedd96f04a5dae160) typo; by Veeupup; 2022-01-04
```

Comment:
We do not update change-log manually.
It is built from commit history. Save a lot work 🤔

Reply:
https://www.grammarly.com/ Grammarly may be helpful. :D